templates/security/notices/usn.html

```diff
@@ -2,125 +2,504 @@
+              <ul class="p-list">
+                {% for package_name, package_title in notice.package_descriptions.items() %}
+                  <li class="p-list__item">
+                    <a href="/security/cves?package={{ package_name }}">{{ package_name }}</a> - {{ package_title }}
+                  </li>
+                {% endfor %}
+              </ul>
+            </div>
+          </div>
+        {% endif %}
+        <div class="p-section">
+          <div class="row">
+            <h2 class="section-heading" id="details" style="scroll-margin-top: 4rem">Details</h2>
+            <div class="js-details-truncated">
+              <p>{{ notice.details | truncate(1000) | safe }}</p>
+            </div>
+            <div class="js-details-all u-hide">
+              <p>{{ notice.details | safe }}</p>
+            </div>
+          </div>
+          {% if notice.details | length > 1000 %}
+            <hr class="p-rule--muted" />
+            <button class="p-button--link js-show-more-details">Show more</button>
+            <button class="p-button--link js-show-less-details u-hide">Show less</button>
+          {% endif %}
+        </div>
+        <div class="p-section">
+          <div class="row">
+            <h2 class="section-heading"
+                id="update-instructions"
+                style="scroll-margin-top: 4rem">Update instructions</h2>
+            <div class="p-notification--information">
+              <div class="p-notification__content">
+                <p class="p-notification__message">
+                  {{ notice.instructions | safe }}<br />
+                  <a href="/security/cves/about#security">Learn more about how to get the fixes.</a>
+                </p>
+              </div>
+            </div>
+
+            {% if notice.attention_banner %}
+              <div class="p-notification--caution">
+                <div class="p-notification__content">
+                  <p class="p-notification__message">{{ notice.attention_banner | safe }}</p>
+                </div>
+              {% endif %}
+            </div>
+            {% if notice.release_packages %}
+              <div class="row">
+                <div class="col-6">
+                  <p>The problem can be corrected by updating your system to the following package versions:</p>
+                </div>
+              </div>
+              <div class="row">
+                <div class="p-section u-cve-table-horizontal-scroll">
+                  <table aria-label="Releases and package versions" class="cve-table u-cve-table-horizontal-scroll__table">
+                    <thead>
+                      <tr>
+                        <th style="width: 12rem;">Ubuntu Release</th>
+                        <th style="width: 18rem;">Package Version</th>
+                      </tr>
+                    </thead>
+                    <tbody>
+                      {% for version in notice.release_packages.__reversed__() %}
```

Comment:
Mentioned in a prev comment, but this was probs a better place to flag the tooltip thing, my bad!